invenio_app_ils/ui/src/invenio_app_ils/pages/backoffice/BackofficeRecordEditor/series/SeriesFormContainer.js

```diff
@@ -0,0 +1,48 @@
+// EditUserDialog.js
```

Comment:
Same here, I would move it with series.

Didn't we say to "kill" the name`Container`?